src/components/shared/ChipList/Wrapper.tsx

```diff
@@ -80,18 +80,30 @@ function ChipWrapper({
                 size="small"
                 variant="outlined"
                 disabled={disabled}
-                onClick={onClick}
+                onClick={
+                    onClick
+                        ? (event) => {
+                              event.stopPropagation();
+                              event.preventDefault();
+                              onClick();
+                          }
+                        : undefined
```

Comment:
Trying to make sure extra actions don't happen when a chip is clicked since it is often inside a table row.